packages/zkdb/src/sdk/signer/nodejs-signer.ts

```diff
@@ -13,7 +13,7 @@ export class NodeSigner implements Signer {
 
   constructor(privateKey: PrivateKey, endpoint: string) {
     this.privateKey = privateKey;
-    this.client = new Client({ network: 'mainnet' });
+    this.client = new Client({ network: 'testnet' });
```

Comment:
Can we stop hardcode this? please make it a configurable instead.

Reply:
Fixed, I changed to passing network, these endpoint and networkId base on user now.

```ts
const signer = new NodeSigner(privateKey, 'testnet')
```